app/Jobs/SendProductDetailToTelegram.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Jobs;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use App\Services\TelegramService;
+
+class SendProductDetailToTelegram implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    private string $photo_url;
+    private string $title;
+    private string $description;
+    private TelegramService $telegram_service;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct($telegram_service ,$photo_url, $title, $description)
```

Comment:
for telegram service use laravel dependency injection ( service container )

Reply:
Added TelegeramService to parameters of handle function .It adds TelegeramService  as a dependency injection.